packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
+-   [*] Social Icons: Fix visibility of inactive icons when used with block based themes in dark mode [https://github.com/WordPress/gutenberg/pull/55398]
```

Comment:
This entry references a different fix and [PR](https://github.com/WordPress/gutenberg/pull/55398) that hasn't been merged yet. Could we update it to reference this PR instead?

Reply:
Whoops, that's me attempting to juggle too many similar changes at once. :D Updated in https://github.com/WordPress/gutenberg/pull/55399/commits/cd0c45fa988142145eb3db591a881e2532ccdbb3.